draft-ietf-scitt-architecture.md

```diff
@@ -367,10 +364,9 @@ The terms "header", "payload", and "to-be-signed bytes" are defined in {{-COSE}}
 
 The term "claim" is defined in {{RFC8392}}.
 
-Append-only Log (Ledger):
+Append-only Log:
 
-: the verifiable append-only data structure that stores Signed Statements in a Transparency Service, often referred to by the synonym Ledger.
-SCITT supports multiple Ledger and Receipt formats to accommodate different Transparency Service implementations, and the proof types associated with different types of Append-only Logs.
+: see Verifiable Data Structure
```

Comment:
"A sequence of entries captured by a Verifiable Data Structure"? The log isn't itself a VDS, right?

Reply:
@achamayou, Could you make a suggestion?